airflow/decorators/__init__.pyi

```diff
@@ -22,11 +22,14 @@
 
 from typing import Any, Callable, Dict, Iterable, List, Mapping, Optional, Union, overload
 
+from kubernetes.client import models as k8s
```

Comment:
hmm this will raise an Import error for folks who don't have k8s library installed but still want to use the other decorators from this file

Reply:
oh wait, this is `.pyi` file

Reply:
Yeah I believe this is OK